nixos/modules/installer/cd-dvd/installation-cd-minimal.nix

```diff
@@ -1,14 +1,17 @@
 # This module defines a small NixOS installation CD.  It does not
 # contain any graphical stuff.
 
-{ ... }:
+{ lib, ... }:
 
 {
-  imports =
-    [ ./installation-cd-base.nix
-    ];
+  imports = [
+    ../../profiles/minimal.nix
+    ./installation-cd-base.nix
+  ];
 
-  isoImage.edition = "minimal";
+  documentation.man.enable = lib.mkOverride 500 true;
 
-  fonts.fontconfig.enable = false;
+  fonts.fontconfig.enable = lib.mkForce false;
```

Comment:
Why not have this in `minimal.nix` as well?

Reply:
This parameter is already triggered by default, if I'm not mistaken. I only added `lib.mkForce` and `lib.mkOverride 500` when there were configuration conflicts.
Still need to add it?